### ISSUE TITLE
withdrawals outside the US, a.k.a. international payouts

### ISSUE THREAD

**Issue by whit537** [gratipay/gratipay.com#126]
Balanced will work for withdrawals to U.S. bank accounts, but not outside the U.S. What do we do here? Suggestions?

### Current Status

 1. **MassPay**&mdash;As a stop-gap while we await the chance to prioritize implementing a more robust solution, we are offering automatic weekly payouts via PayPal. This is risky because PayPal is likely to freeze our account once we get on their radar. If you'd like to participate, email support@gittip.com with your Gittip username, PayPal email address, and the first eight digits of the API key from your Gittip profile page (to confirm your account). PayPal's fee is 2%, capped at $20.
 2. ~~**Balanced**&mdash;Our automated payments provider, [Balanced](https://www.balancedpayments.com/), is working to implement international ACH transfers (IAT). Please see their [roadmap](https://www.balancedpayments.com/international-payments) and voice your interest on [this ticket](https://github.com/balanced/balanced-api/issues/44).~~ *Balanced is shutting down, see #3245.*
 3. **Stripe**&mdash;They've added payouts (à la Balanced), and are known to be working on expanding to the EU. See #3245.
 4. **Trans-Pay**&mdash;The ball is in our court to improve our KYC practices (via #2449?) and circle back. See #417.
 5. **MangoPay**&mdash;They're a European counter-part to Balanced. We would need a European legal entity and bank account. See #891.

### Want to Help?

Give us a PR for multi-currency support. That's a pre-requisite no matter which direction we go.

**Comment by antonsviridenko** [comment]
So, are international SWIFT transfers supported currently or not?

**Comment by rohitpaulk** [comment]
+1 on https://gratipay.freshdesk.com/helpdesk/tickets/1709

**Comment by whit537** [comment]
We have upgraded PayPal from a stop-gap to a first-class payout method—in fact, it's now our *only* payout method (#3567). While not perfect, PayPal does provide more or less global coverage. Therefore, I'm going to take the opportunity to close this old and venerable ticket.

Looking at the other options in the description: the Stripe and MangoPay tickets got closed as part of the Balanced shutdown, because we ended up charting a roll-your-own course instead of using a prefab solution (see https://github.com/gratipay/gratipay.com/issues/3324#issuecomment-120163834 for discussion). Transpay is still on the table, to be revisited once we have [a proper in-house anti-money laundering (AML) program](gratipay/inside.gratipay.com#119). IAT and SWIFT are also options; those were in the works downstream of #3366, which fell through for (again) lack of AML.

Thanks, everyone! :-)
